apache/superset
# fix(select): make to consider the case sensitive in case of d3 format selector

### SUMMARY
d3 format selection widgets should use case sensitive matching for determining uniqueness #17716

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
BEFORE:
![image](https://user-images.githubusercontent.com/47900232/158435840-3de8864f-ccfb-421c-add3-868c7448d35c.png)

AFTER:
![image](https://user-images.githubusercontent.com/47900232/158435054-82729068-6e5c-4c9e-8276-c336c3e00646.png)

### TESTING INSTRUCTIONS

**How to reproduce the bug**

1. Go to 'explore'
2. Create a new 'Time-series Line Chart' (or similar)
3. Click on the 'CUSTOMIZE' tab
4. Try entering the '%b %Y' d3 format for the X-Axis TIME FORMAT (or some other format which matches in a case insensitive way)
5. See that the UI component senses this is a match with '%B %Y' even though these are different

### ADDITIONAL INFORMATION
